### ISSUE TITLE
What are the different versions for?

### ISSUE THREAD

**Issue by Legitti** [LilithRainbows/HabboCustomFlashAIR#6]
No cache doesent cache anything? 60 FPS smoother? what about the others?

**Comment by LilithRainbows** [comment]
Yes, the NoCache version is to disable external resources local disk cache.
60 FPS is to change max FPS count to 60.
I will make a MyFxWidget 60 fps version.

**Comment by Legitti** [comment]
They just patched the AIR with official NFT effect support so I assume MyFxWidget version is no longer needed and I can use just the normal 60fps?
